extensions/ql-vscode/src/vscode-tests/no-workspace/contextual/queryResolver.test.ts

```diff
@@ -93,9 +94,9 @@ describe('queryResolver', () => {
     getPrimaryDbschemeSpy = sinon.stub();
     return proxyquire('../../../contextual/queryResolver', {
       'fs-extra': {
+        ensureDirSync,
```

Comment:
On line 9, there is:

```
const proxyquire = pq.noPreserveCache().noCallThru();
```

I don't think `noCallThru` is necessary. If you remove that, do you still need to have this line?